content/utmt/audio/adding sounds and music/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Adding Sounds and Music 
+
+resources:
+  - name: soundsAdd
+    src: "soundsAdd.png"
+    title: Adding a new sound
+  - name: embeddedAudioAdd
+    src: "embeddedAudioAdd.png"
+    title: Adding embedded audio
+  - name: newEmbeddedAudio
+    src: "newEmbeddedAudio.png"
+    title: A new Embedded Audio asset
+  - name: newSound
+    src: "newSound.png"
+    title: A new Sound's properties	
+---
+Sounds can be either loaded internally (a WAV file embedded in the `data.win`) or externally (an Ogg Vorbis file placed in the game's folder).
+{{<tabs "source">}}
+{{<tab "Internal">}}
```

Comment:
step 1 should mirror step 1 in music. Aka either have both mentioning a reminder to have the correct audio format, or none.